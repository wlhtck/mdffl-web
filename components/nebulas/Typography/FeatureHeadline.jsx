import withStyles from '../withStyles';
import styles from './styles';

const FeatureHeadline = withStyles(styles({
  family: 'raleway', weight: 700, size: '36px',
}), 'h2');

export default FeatureHeadline;
