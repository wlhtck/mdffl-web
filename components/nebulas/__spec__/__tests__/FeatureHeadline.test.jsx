import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import FeatureHeadline from '../../Typography/FeatureHeadline';

describe('nebulas/Typography/FeatureHeadline', () => {
  it('renders a <Headine />', () => {
    expect(mount(<FeatureHeadline>Mission Statement</FeatureHeadline>)).toMatchSnapshot();
  });
});
