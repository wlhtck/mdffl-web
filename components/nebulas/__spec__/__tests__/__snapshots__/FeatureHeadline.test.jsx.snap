// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nebulas/Typography/FeatureHeadline renders a <Headine /> 1`] = `
.c0 {
  color: #303133;
  font-family: 'Raleway',sans-serif;
  font-weight: 700;
  font-size: 36px;
  text-align: left;
  margin: 0 0 32px;
}

<withStyles>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "withStyles-sc-8qn4cd-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            [Function],
          ],
        },
        "displayName": "withStyles",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "withStyles-sc-8qn4cd-0",
        "target": "h2",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <h2
      className="c0"
    >
      Mission Statement
    </h2>
  </StyledComponent>
</withStyles>
`;
